ui/package.json

```diff
@@ -12,53 +12,58 @@
     "test": "jest",
     "lint": "eslint --cache --format codeframe --ext mjs,jsx,js src test"
   },
+  "comment": "IMPORTANT! date-fns is pinned to 1.3.x due to https://github.com/mui-org/material-ui-pickers/issues/1440",
+  "comment2": "IMPORTANT! react-virtualized is held at 9.21.1 due to https://github.com/bvaughn/react-virtualized/issues/1226 breaking scrolling to anchors on ListRules",
```

Comment:
Per our agreement, let's pin react, react-dom, @hot-loader/react-dom to an earlier version where the scroll feature worked.

Reply:
While trying to verify this I realized that my earlier statement was wrong. This actually works fine with the latest react, as long as we don't upgrade `react-virtualized`. I'll pin that instead.